subsys/bluetooth/host/hci_core.c

```diff
@@ -914,6 +914,36 @@ static int hci_le_read_remote_features(struct bt_conn *conn)
 	return 0;
 }
 
+#if defined(CONFIG_BT_REMOTE_VERSION)
+int hci_le_read_remote_version(struct bt_conn *conn)
```

Comment:
Move this into `conn.c`. There's no reason to have it in hci_core.c and then you have to export it through a header file instead of it being static.